bindgen-tests/tests/expectations/tests/macro-expr-basic.rs

```diff
@@ -8,7 +8,7 @@
 pub const FOO: u32 = 1;
 pub const BAR: u32 = 4;
 pub const BAZ: u32 = 5;
-pub const MIN: i64 = -9223372036854775808;
+pub const MIN: u64 = 9223372036854775808;
```

Comment:
This changed since previously numbers were cast to `i64` internally in `cexpr`. Now that `i128` is used, it actually fits in the positive part of `i128`, resulting in `u64` being used.

Reply:
Oh OK I get it now. So if the original value is positive you try to fit in in a unsigned integer.